app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -41,27 +44,31 @@
 
 import co.copperhead.pdfviewer.fragment.DocumentPropertiesFragment;
 import co.copperhead.pdfviewer.fragment.JumpToPageFragment;
+import co.copperhead.pdfviewer.fragment.PasswordPromptFragment;
 
-public class PdfViewer extends Activity {
+public class PdfViewer extends AppCompatActivity {
```

Comment:
I'm okay with using AppCompat now since Google is essentially making the support library mandatory with Android P so that should make this easier. They're even going to be deprecating using native fragments, so there's no point in avoiding the support library.

Reply:
Their documentation is still presenting it as primarily for backwards compatibility but they're essentially hoisting everything they can out of the platform into it now.